AlertaDengue/dados/tests.py

```diff
@@ -1,4 +1,85 @@
 # coding=utf-8
+import json
+
+from django.core.urlresolvers import reverse
 from django.test import TestCase
 
-# Create your tests here.
+class TestAlertaPageView(TestCase):
+    def setUp(self):
+        # TODO: Create fixtures to prevent test fails in case of data changing
```

Comment:
I think we finish the fixtures before merging so that we are independent of current test dataset data.